version.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "https://raw.githubusercontent.com/AArnott/Nerdbank.GitVersioning/master/src/NerdBank.GitVersioning/version.schema.json",
+  "version": "0.15.1.0",
```

Comment:
:memo: This is capable of calculating unique version numbers for each commit on build, but for now I've hard-coded all four digits of the version number to match the behavior prior to this pull request. The partial adoption of Nerdbank.GitVersioning causes the `ThisAssembly` class to be generated during the build, which is used by `InitializeVersion` to provide version information now that the nuspec file is generated after the assembly has already been built.

The behavior *change* introduced by this involves the `[AssemblyVersion]` and `[AssemblyFileVersion]` attributes, which now read `0.15.0.0` and `0.15.1.0`, respectively.